tests/utils/stats_fetcher.go

```diff
@@ -27,7 +27,7 @@ type StatHistograms map[string][]float64
 
 const (
 	// Path with filtering for ESPv2 stats.
-	ESpv2FiltersStatsPath = "/stats?format=json&usedonly&filter=http.ingress_http.(path_matcher|backend_auth|service_control|backend_routing)"
```

Comment:
Do we have any stats integration tests for backend routing that need to be modified for path_rewrite?

Reply:
Yes, there is only on in TestStatistics.  It has been modified.